docs/source/algorithms/FindPeaksMD-v1.rst

```diff
@@ -83,8 +83,12 @@ First calculate the :math:`\textbf{Q}_{lab}` using
 
 .. math:: \theta = \cos^{-1}(1-\frac{|\textbf{Q}_{sample}|^2}{2k^2})
 
+where :math:`\theta` is from 0 to :math:`\pi`
+
 .. math:: \phi = \sin^{-1}(-\textbf{Q}_{sample}^y \sin(\theta)/k)
 
+where :math:`\phi` is from :math:`-\pi/2` to :math:`\pi/2`
+
```

Comment:
Need to explicitly tell that this means that you will get a detector position on the left hand side of the beam, even if you don't have a detector there.